source/kubernetes/quickstart.rst

```diff
@@ -193,91 +173,60 @@ to interact with your Catalyst Cloud Kubernetes cluster using the command line.
 As the Kubernetes dashboard of our cluster is not directly accessible from the Internet,
 we will use ``kubectl`` to gain access to the Kubernetes dashboard.
 
-First, run the following commands to create the **kubeconfig** file.
+First, ensure you have downloaded the **kubeconfig** file from the Catalyst Cloud
+dashboard using the **Download Kubeconfig** button.
+
+.. note::
+
+  Your browser may download the **kubeconfig** file to another location. Specific
+  browser download locations are out of scope for this document, you can also locate these
+  within your browser.
+
 The kubeconfig file contains the required metadata used to authenticate
 with the Kubernetes cluster.
 
 .. tabs::
 
     .. group-tab:: Linux / macOS
 
-      .. note::
-
-        In the below examples, the kubeconfig file will be created in the current folder
-        of your terminal session.
-
-        If you wish to save the configuration to a different location, replace ``$(pwd)``
-        with your preferred destination folder.
-
-      The following command will create the kubeconfig in the target directory
-      with the filename ``config``.
+      In a command line terminal environment (such as ``bash``) export the ``KUBECONFIG``
+      environment variable, to configure ``kubectl`` to connect to your cluster.
 
       .. code-block:: bash
 
-        openstack coe cluster config quickstart-cluster --use-keystone --dir "$(pwd)"
+        export KUBECONFIG="${HOME}/Downloads/quickstart1_kubeconfig"
 
-      Now export the ``KUBECONFIG`` environment variable, to configure ``kubectl``
-      to connect to your cluster.
+      Alternatively, if you have no other Kubernetes clusters, you could move the downloaded file `quickstart1_kubeconfig` to the default location for ``kubectl``, in `$HOME/.kube/config`.
 
       .. code-block:: bash
 
-        export KUBECONFIG="$(pwd)/config"
-
-    .. group-tab:: Windows (PowerShell)
-
-      .. note::
-
-        In the below examples, the kubeconfig file will be created in the current folder
-        of your terminal session.
+        mv ${HOME}/Downloads/quickstart1_kubeconfig ${HOME}/.kube/config
```

Comment:
Ensure `${HOME}/.kube` exists. Also, quote things to protect against spaces causing problems.

```suggestion
        mkdir -p "${HOME}/.kube/config" && mv "${HOME}/Downloads/quickstart1_kubeconfig" "${HOME}/.kube/config"
```

Reply:
This suggestion was slightly incorrect. Just updated it.